test/tests/functional/pbs_ralter.py

```diff
@@ -2287,6 +2287,35 @@ def test_alter_standing_resv_future_occrs(self):
         rid2, start, end = self.submit_and_confirm_reservation(
             3000, 1800, select="2:ncpus=4")
 
+    def test_alter_start_standing_resv_future_occrs(self):
+        """
+        Test that when duration of a confirmed standing reservation is
+        altered, only the upcoming occurence changes and not all occurences
+        are modified.
+        """
```

Comment:
Don't both the tests have the same function name?

Reply:
oops